### ISSUE TITLE
2024-12-05, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#11]
# Today's Arxiv Papers

(1) [From Individual to Society: A Survey on Social Simulation Driven by Large Language Model-based Agents](http://arxiv.org/abs/2412.03563v1), 2024-12-04

(2) [SPICE: Smart Projection Interface for Cooking Enhancement](http://arxiv.org/abs/2412.03551v1), 2024-12-04

(3) [Evaluating Gender Bias Transfer between Pre-trained and Prompt-Adapted Language Models](http://arxiv.org/abs/2412.03537v1), 2024-12-04

(4) [A Review on Scientific Knowledge Extraction using Large Language Models in Biomedical Sciences](http://arxiv.org/abs/2412.03531v1), 2024-12-04

(5) [FANAL -- Financial Activity News Alerting Language Modeling Framework](http://arxiv.org/abs/2412.03527v1), 2024-12-04

(6) [You're (Not) My Type -- Can LLMs Generate Feedback of Specific Types for Introductory Programming Tasks?](http://arxiv.org/abs/2412.03516v1), 2024-12-04

(7) [Training-Free Mitigation of Language Reasoning Degradation After Multimodal Instruction Tuning](http://arxiv.org/abs/2412.03467v1), 2024-12-04

(8) [From Words to Workflows: Automating Business Processes](http://arxiv.org/abs/2412.03446v1), 2024-12-04

(9) [RedStone: Curating General, Code, Math, and QA Data for Large Language Models](http://arxiv.org/abs/2412.03398v1), 2024-12-04

(10) [WiS Platform: Enhancing Evaluation of LLM-Based Multi-Agent Systems Through Game-Based Analysis](http://arxiv.org/abs/2412.03359v1), 2024-12-04

(11) [Improving Linguistic Diversity of Large Language Models with Possibility Exploration Fine-Tuning](http://arxiv.org/abs/2412.03343v1), 2024-12-04

(12) [AI-Driven Day-to-Day Route Choice](http://arxiv.org/abs/2412.03338v1), 2024-12-04

(13) [Intent-driven In-context Learning for Few-shot Dialogue State Tracking](http://arxiv.org/abs/2412.03270v1), 2024-12-04

(14) [DynamicControl: Adaptive Condition Selection for Improved Text-to-Image Generation](http://arxiv.org/abs/2412.03255v1), 2024-12-04

(15) [Alignment at Pre-training! Towards Native Alignment for Arabic LLMs](http://arxiv.org/abs/2412.03253v1), 2024-12-04

(16) [Controlling the Mutation in Large Language Models for the Efficient Evolution of Algorithms](http://arxiv.org/abs/2412.03250v1), 2024-12-04

(17) [AIM: Adaptive Inference of Multi-Modal LLMs via Token Merging and Pruning](http://arxiv.org/abs/2412.03248v1), 2024-12-04

(18) [Does Safety Training of LLMs Generalize to Semantically Related Natural Prompts?](http://arxiv.org/abs/2412.03235v1), 2024-12-04

(19) [Linq-Embed-Mistral Technical Report](http://arxiv.org/abs/2412.03223v1), 2024-12-04

(20) [Survey of different Large Language Model Architectures: Trends, Benchmarks, and Challenges](http://dx.doi.org/10.1109/ACCESS.2024.3482107), 2024-12-04

(21) [ClusterKV: Manipulating LLM KV Cache in Semantic Space for Recallable Compression](http://arxiv.org/abs/2412.03213v1), 2024-12-04

(22) [U-MATH: A University-Level Benchmark for Evaluating Mathematical Skills in LLMs](http://arxiv.org/abs/2412.03205v1), 2024-12-04

(23) [Beyond Questions: Leveraging ColBERT for Keyphrase Search](http://arxiv.org/abs/2412.03193v1), 2024-12-04

(24) [Weighted-Reward Preference Optimization for Implicit Model Fusion](http://arxiv.org/abs/2412.03187v1), 2024-12-04

(25) [Automatic detection of diseases in Spanish clinical notes combining medical language models and ontologies](http://arxiv.org/abs/2412.03176v1), 2024-12-04

(26) [LLM-Twin: A Generated-Persona Approach for Survey Pre-Testing](http://arxiv.org/abs/2412.03162v1), 2024-12-04

(27) [Byte BPE Tokenization as an Inverse string Homomorphism](http://arxiv.org/abs/2412.03160v1), 2024-12-04

(28) [Large Language Models show both individual and collective creativity comparable to humans](http://arxiv.org/abs/2412.03151v1), 2024-12-04

(29) [Fine-Grained Behavior Simulation with Role-Playing Large Language Model on Social Media](http://arxiv.org/abs/2412.03148v1), 2024-12-04

(30) [Unifying KV Cache Compression for Large Language Models with LeanKV](http://arxiv.org/abs/2412.03131v1), 2024-12-04

(31) [Robust Multi-bit Text Watermark with LLM-based Paraphrasers](http://arxiv.org/abs/2412.03123v1), 2024-12-04

(32) [CredID: Credible Multi-Bit Watermark for Large Language Models Identification](http://arxiv.org/abs/2412.03107v1), 2024-12-04

(33) [ChatTS: Aligning Time Series with LLMs via Synthetic Data for Enhanced Understanding and Reasoning](http://arxiv.org/abs/2412.03104v1), 2024-12-04

(34) [TOOL-ED: Enhancing Empathetic Response Generation with the Tool Calling Capability of LLM](http://arxiv.org/abs/2412.03096v1), 2024-12-04

(35) [Revolve: Optimizing AI Systems by Tracking Response Evolution in Textual Optimization](http://arxiv.org/abs/2412.03092v1), 2024-12-04

(36) [Mimir: Improving Video Diffusion Models for Precise Text Understanding](http://arxiv.org/abs/2412.03085v1), 2024-12-04

(37) [ASR-EC Benchmark: Evaluating Large Language Models on Chinese ASR Error Correction](http://arxiv.org/abs/2412.03075v1), 2024-12-04

(38) [Human Variability vs. Machine Consistency: A Linguistic Analysis of Texts Generated by Humans and Large Language Models](http://arxiv.org/abs/2412.03025v1), 2024-12-04

(39) [Advancing Conversational Psychotherapy: Integrating Privacy, Dual-Memory, and Domain Expertise with Large Language Models](http://arxiv.org/abs/2412.02987v1), 2024-12-04

(40) [Surveying the Effects of Quality, Diversity, and Complexity in Synthetic Data From Large Language Models](http://arxiv.org/abs/2412.02980v1), 2024-12-04

(41) [Theoretical limitations of multi-layer Transformer](http://arxiv.org/abs/2412.02975v1), 2024-12-04

(42) [Curriculum-style Data Augmentation for LLM-based Metaphor Detection](http://arxiv.org/abs/2412.02956v1), 2024-12-04

(43) [Video LLMs for Temporal Reasoning in Long Videos](http://arxiv.org/abs/2412.02930v1), 2024-12-04

(44) [Single-Cell Omics Arena: A Benchmark Study for Large Language Models on Cell Type Annotation Using Single-Cell Data](http://arxiv.org/abs/2412.02915v1), 2024-12-03

(45) [Does Few-Shot Learning Help LLM Performance in Code Synthesis?](http://arxiv.org/abs/2412.02906v1), 2024-12-03

(46) [Enhancing Trust in Large Language Models with Uncertainty-Aware Fine-Tuning](http://arxiv.org/abs/2412.02904v1), 2024-12-03

(47) [MLD-EA: Check and Complete Narrative Coherence by Introducing Emotions and Actions](http://arxiv.org/abs/2412.02897v1), 2024-12-03

(48) [Removing Spurious Correlation from Neural Network Interpretations](http://arxiv.org/abs/2412.02893v1), 2024-12-03

(49) [Patchfinder: Leveraging Visual Language Models for Accurate Information Retrieval using Model Uncertainty](http://arxiv.org/abs/2412.02886v1), 2024-12-03

(50) [TDD-Bench Verified: Can LLMs Generate Tests for Issues Before They Get Resolved?](http://arxiv.org/abs/2412.02883v1), 2024-12-03

(51) [A Novel Compact LLM Framework for Local, High-Privacy EHR Data Applications](http://arxiv.org/abs/2412.02868v1), 2024-12-03

(52) [Measuring Bias of Web-filtered Text Datasets and Bias Propagation Through Training](http://arxiv.org/abs/2412.02857v1), 2024-12-03

(53) [RARE: Retrieval-Augmented Reasoning Enhancement for Large Language Models](http://arxiv.org/abs/2412.02830v1), 2024-12-03

(54) [Minimization of Boolean Complexity in In-Context Concept Learning](http://arxiv.org/abs/2412.02823v1), 2024-12-03

(55) [CNNSum: Exploring Long-Conext Summarization with Large Language Models in Chinese Novels](http://arxiv.org/abs/2412.02819v1), 2024-12-03

(56) [Unleashing GHOST: An LLM-Powered Framework for Automated Hardware Trojan Design](http://arxiv.org/abs/2412.02816v1), 2024-12-03

(57) [Flattering to Deceive: The Impact of Sycophantic Behavior on User Trust in Large Language Model](http://arxiv.org/abs/2412.02802v1), 2024-12-03

(58) [An Evolutionary Large Language Model for Hallucination Mitigation](http://arxiv.org/abs/2412.02790v1), 2024-12-03

(59) [Exploring the Potential of Llama Models in Automated Code Refinement: A Replication Study](http://arxiv.org/abs/2412.02789v1), 2024-12-03

(60) [Hybrid-SQuAD: Hybrid Scholarly Question Answering Dataset](http://arxiv.org/abs/2412.02788v1), 2024-12-03

(61) [FathomGPT: A Natural Language Interface for Interactively Exploring Ocean Science Data](http://dx.doi.org/10.1145/3654777.3676462), 2024-12-03

(62) [How to quantify the coherence of a set of beliefs](http://arxiv.org/abs/2412.02777v1), 2024-12-03

(63) [Hacking CTFs with Plain Agents](http://arxiv.org/abs/2412.02776v1), 2024-12-03

(64) [Optimizing Large Language Models for Turkish: New Methodologies in Corpus Selection and Training](http://dx.doi.org/10.1109/ASYU62119.2024.10757019), 2024-12-03

(65) [Drawing Pandas: A Benchmark for LLMs in Generating Plotting Code](http://arxiv.org/abs/2412.02764v1), 2024-12-03

(66) [Cosmos-LLaVA: Chatting with the Visual Cosmos-LLaVA: Görselle Sohbet Etmek](http://dx.doi.org/10.1109/IDAP64064.2024.10710874), 2024-12-03

(67) [T-REG: Preference Optimization with Token-Level Reward Regularization](http://arxiv.org/abs/2412.02685v1), 2024-12-03

(68) [Mind the Gap: Examining the Self-Improvement Capabilities of Large Language Models](http://arxiv.org/abs/2412.02674v1), 2024-12-03

(69) [CPP-UT-Bench: Can LLMs Write Complex Unit Tests in C++?](http://arxiv.org/abs/2412.02735v1), 2024-12-03

(70) [LLM-Enhanced Path Planning: Safe and Efficient Autonomous Navigation with Instructional Inputs](http://arxiv.org/abs/2412.02655v1), 2024-12-03

(71) [QA-TOOLBOX: Conversational Question-Answering for process task guidance in manufacturing](http://arxiv.org/abs/2412.02638v1), 2024-12-03

(72) [Time-Reversal Provides Unsupervised Feedback to LLMs](http://arxiv.org/abs/2412.02626v2), 2024-12-03

(73) [Improving Dynamic Object Interactions in Text-to-Video Generation with AI Feedback](http://arxiv.org/abs/2412.02617v1), 2024-12-03

(74) [AV-Odyssey Bench: Can Your Multimodal LLMs Really Understand Audio-Visual Information?](http://arxiv.org/abs/2412.02611v1), 2024-12-03

(75) [Interpretable Company Similarity with Sparse Autoencoders](http://arxiv.org/abs/2412.02605v1), 2024-12-03

(76) [CEGI: Measuring the trade-off between efficiency and carbon emissions for SLMs and VLMs](http://arxiv.org/abs/2412.02602v1), 2024-12-03

(77) [PrefixLLM: LLM-aided Prefix Circuit Design](http://arxiv.org/abs/2412.02594v1), 2024-12-03

(78) [OCR Hinders RAG: Evaluating the Cascading Impact of OCR on Retrieval-Augmented Generation](http://arxiv.org/abs/2412.02592v1), 2024-12-03

(79) [Explainable CTR Prediction via LLM Reasoning](http://arxiv.org/abs/2412.02588v1), 2024-12-03

(80) [Semantic Tokens in Retrieval Augmented Generation](http://arxiv.org/abs/2412.02563v1), 2024-12-03

(81) [Patent-CR: A Dataset for Patent Claim Revision](http://arxiv.org/abs/2412.02549v1), 2024-12-03

(82) [LLMForecaster: Improving Seasonal Event Forecasts with Unstructured Textual Data](http://arxiv.org/abs/2412.02525v1), 2024-12-03

(83) [DP-2Stage: Adapting Language Models as Differentially Private Tabular Data Generators](http://arxiv.org/abs/2412.02467v1), 2024-12-03

(84) [Gracefully Filtering Backdoor Samples for Generative Large Language Models without Retraining](http://arxiv.org/abs/2412.02454v1), 2024-12-03

(85) [GerPS-Compare: Comparing NER methods for legal norm analysis](http://arxiv.org/abs/2412.02427v1), 2024-12-03

(86) [A Multi-Agent Framework for Extensible Structured Text Generation in PLCs](http://arxiv.org/abs/2412.02410v1), 2024-12-03

(87) [ScImage: How Good Are Multimodal Large Language Models at Scientific Text-to-Image Generation?](http://arxiv.org/abs/2412.02368v1), 2024-12-03

(88) [Large Multimodal Agents for Accurate Phishing Detection with Enhanced Token Optimization and Cost Reduction](http://arxiv.org/abs/2412.02301v1), 2024-12-03

(89) [A Comprehensive Evaluation of Large Language Models on Aspect-Based Sentiment Analysis](http://arxiv.org/abs/2412.02279v1), 2024-12-03

(90) [MediaSpin: Exploring Media Bias Through Fine-Grained Analysis of News Headlines](http://arxiv.org/abs/2412.02271v1), 2024-12-03

(91) [Connecting Large Language Models with Blockchain: Advancing the Evolution of Smart Contracts from Automation to Intelligence](http://arxiv.org/abs/2412.02263v1), 2024-12-03

(92) [Compressing KV Cache for Long-Context LLM Inference with Inter-Layer Attention Similarity](http://arxiv.org/abs/2412.02252v1), 2024-12-03

(93) [BANER: Boundary-Aware LLMs for Few-Shot Named Entity Recognition](http://arxiv.org/abs/2412.02228v1), 2024-12-03

(94) [Unlocking Tuning-Free Few-Shot Adaptability in Visual Foundation Models by Recycling Pre-Tuned LoRAs](http://arxiv.org/abs/2412.02220v1), 2024-12-03

(95) [DataLab: A Unified Platform for LLM-Powered Business Intelligence](http://arxiv.org/abs/2412.02205v2), 2024-12-03

(96) [3D representation in 512-Byte:Variational tokenizer is the key for autoregressive 3D generation](http://arxiv.org/abs/2412.02202v1), 2024-12-03

(97) [LayoutVLM: Differentiable Optimization of 3D Layout via Vision-Language Models](http://arxiv.org/abs/2412.02193v1), 2024-12-03

(98) [Keeping Experts in the Loop: Expert-Guided Optimization for Clinical Data Classification using Large Language Models](http://arxiv.org/abs/2412.02173v1), 2024-12-03

(99) [Jailbreak Defense in a Narrow Domain: Limitations of Existing Methods and a New Transcript-Classifier Approach](http://arxiv.org/abs/2412.02159v1), 2024-12-03

(100) [CausalMob: Causal Human Mobility Prediction with LLMs-derived Human Intentions toward Public Events](http://arxiv.org/abs/2412.02155v1), 2024-12-03
